CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing
+
+We would ❤️ for you to contribute to Appwrite and help make it better! We want contributing to Appwrite to be fun, enjoyable, and educational for anyone and everyone. All contributions are welcome, including issues, new docs as well as updates and tweaks, blog posts, workshops, and more.
+
+## How to Start?
+
+If you are worried or don’t know where to start, check out our next section explaining what kind of help we could use and where can you get involved. You can reach out with questions on our [Discord](https://discord.gg/GSeTUeA) server. You can also submit an issue and a maintainer can guide you!
+
+## Code of Conduct
+
+Help us keep Appwrite open and inclusive. Please read and follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
+
+## Submit a Pull Request 🚀
+
+Branch naming convention is as follows
+
+`TYPE-ISSUE_ID-DESCRIPTION`
+
+example:
+
+```
+doc-548-submit-a-pull-request-section-to-contribution-guide
+```
+
+When `TYPE` can be:
+
+- **feat** - is a new feature
+- **doc** - documentation only changes
+- **cicd** - changes related to CI/CD system
+- **fix** - a bug fix
+- **refactor** - code change that neither fixes a bug nor adds a feature
+
+**All PRs must include a commit message with the description of the changes made!**
+
+For the initial start, fork the project and use git clone command to download the repository to your computer. A standard procedure for working on an issue would be to:
+
+1. `git pull`, before creating a new branch, pull the changes from upstream. Your master needs to be up to date.
+
+```
+$ git pull
+```
+
+2. Create new branch from `master` like: `doc-548-submit-a-pull-request-section-to-contribution-guide`
+
+```
+$ git checkout -b [name_of_your_new_branch]
+```
+
+3. Work - commit - repeat ( be sure to be in your branch )
+4. Push changes to GitHub
+
+```
+$ git push origin [name_of_your_new_branch]
+```
+
+6. Submit your changes for review. If you go to your repository on GitHub, you'll see a `Compare & pull request` button. Click on that button.
+7. Start a Pull Request (PR) by clicking on `Create pull request`. Make sure to update the PR description following the template provided.
+8. Wait for a code review.
+9. If you need to make changes based on feedback, make sure to re-request a review from your reviewer after you've made the necessary changes.
+
+![Re-Request a Review](https://docs.github.com/assets/cb-4714/images/help/pull_requests/request-re-review.png)
+
+10. After approval, your PR will be merged.
+11. You can delete your branch after it has been merged.
```

Comment:
I made some tweaks here compared to:

https://github.com/appwrite/appwrite/blob/44ca7d9459dba9b21760e2934decbfd41f26c1ba/CONTRIBUTING.md?plain=1#L75-L81

Reply:
But also, they'd be forking, right? This would delete their fork's branch.

Reply:
Yes, correct.